<?xml version="1.0" encoding="UTF-8"?>
<!--
    Copyright 2005-2006 The Apache Software Foundation or its licensors, as applicable

    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at

       http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

    See the License for the specific language governing permissions and
    limitations under the License.
-->
<!DOCTYPE Test SYSTEM "test.dtd">
<!--
Author: Petr Ivanov
Version: $Revision: 1.2 $
-->

<Test
    ID="ExceptionDescribeTest"
    date-of-creation="12-11-2004"
    timeout="1"
    >   
        <Author value="Gregory Shimansky, Petr Ivanov"/>
        <Description>
            <TestedMethod name="ExceptionDescribe()"/>
            Native code throws an exception and uses ExceptionDescribe
            Test should not break vm.

            Covered assertions assert_func079 assert_func073
        </Description>
        <Keyword name="functional"/>    
        <Source name="ExceptionDescribeTest.java"/>
            
        <Runner ID="Runtime">
            <Param name="toRun" value="org.apache.harmony.vts.test.vm.jni.exceptions.ExceptionDescribeTest"/>
        </Runner>
</Test>
